jck/jtrunner/JavaTestRunner.java

```diff
@@ -649,7 +649,9 @@ public static boolean jckConfigurationForMultijvmWithNoAgent() throws Exception
 
 			// The jplisLivePhase and Robot available settings are rejected if placed higher up in the .jtb file
 			if ( tests.contains("api/java_awt") || tests.contains("api/javax_swing") || tests.equals("api") ) {
-				fileContent += "set jck.env.runtime.awt.robotAvailable " + robotAvailable + ";\n";
+				if ( robotAvailable == "Yes" ) {
```

Comment:
Is this check needed?  wouldn't it just add the line `set jck.env.runtime.awt.robotAvailable No` in the 2 cases where its set to no (alpine-linux / zos)?